Add parentheses surrounding cross-references

nedmsmith opened the pull request:
Updates section and table references that don't have surrounding parens.

See issue #286

yogeshbdeshpande reviewed (approved):
LGTM!

thomas-fossati commented on draft-ietf-rats-corim.md:
```diff
@@ -286,29 +286,29 @@ constraints must be followed when creating or validating a CoRIM map.
-  in {{sec-corim-tags}}
+  in ({{sec-corim-tags}})
 
 * `dependent-rims` (index 2): One or more services supplying additional,
   possibly dependent, manifests or related files. Described in
-  {{sec-corim-locator-map}}
+  ({{sec-corim-locator-map}})
 
 * `profile` (index 3): An optional profile identifier for the tags contained in
   this CoRIM.  The profile MUST be understood by the CoRIM processor.  Failure
   to recognize the profile identifier MUST result in the rejection of the
   entire CoRIM.  If missing, the profile defaults to DICE.
-  Described in {{sec-corim-profile-types}}
+  Described in ({{sec-corim-profile-types}})
 
 * `rim-validity` (index 4): Specifies the validity period of the CoRIM.
-  Described in {{sec-common-validity}}
+  Described in ({{sec-common-validity}})
 
 * `entities` (index 5): A list of entities involved in a CoRIM life-cycle.
-  Described in {{sec-corim-entity}}
+  Described in ({{sec-corim-entity}})
 
 * `$$corim-map-extension`: This CDDL socket is used to add new information
-  structures to the `corim-map`.  See {{sec-iana-corim}}.
+  structures to the `corim-map`.  See ({{sec-iana-corim}}).
```
These look spurious.  They'd render as:

> Described in (Section 1.2.3)

which is not what we want.  Instead, they should read as:

> Described in Section 1.2.3.

thomas-fossati commented on draft-ietf-rats-corim.md:
```diff
@@ -347,7 +347,7 @@ The following describes each child element of this type.
 
 * `thumbprint` (index 1): expected digest of the resource referenced by `href`.
-  See {{sec-common-hash-entry}}.
+  See ({{sec-common-hash-entry}}).
```
ditto

thomas-fossati commented on draft-ietf-rats-corim.md:
```diff
@@ -457,10 +457,10 @@ This ensures the consumer is able to identify credentials used to authenticate i
 * `signer` (index 0): Information about the entity that signs the CoRIM.
-  Described in {{sec-corim-signer}}
+  Described in ({{sec-corim-signer}})
 
 * `signature-validity` (index 1): Validity period for the CoRIM. Described in
-  {{sec-common-validity}}
+  ({{sec-common-validity}})
```
ditto

thomas-fossati commented on draft-ietf-rats-corim.md:
```diff
@@ -526,13 +526,13 @@ The following describes each member of the `concise-mid-tag` map.
+  identification information for the CoMID. Described in ({{sec-comid-tag-id}}).
 
 * `entities` (index 2): Provides information about one or more organizations
-  responsible for producing the CoMID tag. Described in {{sec-comid-entity}}.
+  responsible for producing the CoMID tag. Described in ({{sec-comid-entity}}).
 
-* `linked-tags` (index 3): A list of one or more `linked-tag-map` (described in
-  {{sec-comid-linked-tag}}), providing typed relationships between this and
+* `linked-tags` (index 3): A list of one or more `linked-tag-map`. Described in
+  ({{sec-comid-linked-tag}}), providing typed relationships between this and
```
ditto, and below too.

nedmsmith commented:
See if commit https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/300/commits/1372934797b8e6ca7018730564e1a430a74fe8f4 addresses your concerns.

thomas-fossati commented:
> See if commit [1372934](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/commit/1372934797b8e6ca7018730564e1a430a74fe8f4) addresses your concerns.

Thanks for the changes!
